### DISCUSSION TITLE
kafkaconnect to use sink and source plugin ( backup and restore ) using azure blob

### DISCUSSION THREAD

**Discussion by Veljen** [strimzi/strimzi-kafka-operator#9556] [Q&A]
Hi,

Is it possible to use kafkaconnect sink to backup the topics to the azure blob storage and restore it using source connector in same or another cluster? 

It seems source plugin is not opensource. is there any opensource plugin we can use it in strimzi and achieve this ?  @scholzj  Need your advice.

**Comment by scholzj** [comment]
You can use Connect to push the messages from Kafka to other storage systems. But the recovery is not always straight-forward. So it depends if you want to be able to quickly and easily recover or if you just want to have the data somewhere for archiving purposes. That said, I have no experience with Azure Blob Storage, so cannot advice on that.
